pkgs/development/interpreters/python/hooks/setuptools-rust-hook.sh

```diff
@@ -0,0 +1,18 @@
+echo "Sourcing setuptools-rust-hook"
+
+setuptoolsRustSetup() {
+    # This can work only if rustPlatform.cargoSetupHook is also included
+    if ! command -v cargoSetupPostPatchHook >/dev/null; then
```

Comment:
```suggestion
    if ! type cargoSetupPostPatchHook 2>/dev/null >/dev/null; then
```
command only really works correct with entries in PATH and has rather weird behaviors with shell functions. type is a bit more robust

Reply:
Good point. Applied with a small tweak to the shell.